.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,38 @@
+
+      - name: Backend dependencies
+        run: cd backend && npm install
+
+
+      - name: Run unit tests with Jest
+        run: cd backend && npm run test:coverage
+
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarqube-scan-action@v5.1.0
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)